src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -479,204 +633,264 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
                 return;
             }
 
-            var elem = self._mediaElement;
+            const elem = this.#mediaElement;
             if (!elem) {
                 return;
             }
 
             // https://msdn.microsoft.com/en-us/library/hh772507(v=vs.85).aspx
 
-            var elemAudioTracks = elem.audioTracks || [];
-            console.debug('found ' + elemAudioTracks.length + ' audio tracks');
+            const elemAudioTracks = elem.audioTracks || [];
+            console.debug(`found ${elemAudioTracks.length} audio tracks`);
 
-            for (i = 0, length = elemAudioTracks.length; i < length; i++) {
+            for (const [i, audioTrack] of elemAudioTracks.entries()) {
```

Comment:
On webOS 1.2, `elemAudioTracks` (`AudioTrackList`) doesn't have `entries`.
I cannot get to here in browsers, but probably the same for them.

Reply:
That's probably a missing polyfill there

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/entries

Reply:
The problem is that `AudioTrackList` is not `Array`.
I had to revert the same for `HTMLOptionsCollection` #1279

Reply:
Oh okay I can work with that, `Array.from()` does the job

Reply:
There you go